src/code-templates/services/order-service/jest.config.js

```diff
@@ -99,7 +99,20 @@ module.exports = {
   // projects: undefined,
 
   // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
+  reporters: [
+    "default",
+    [
+      "@jest-performance-reporter/core",
+      {
+        "errorAfterMs": 1000,
+        "warnAfterMs": 500,
+        "logLevel": "warn",
+        "maxItems": 5,
+        "jsonReportPath": "performance-reports/performance-report.json",
+        "csvReportPath": "performance-reports/performance-report.csv"
```

Comment:
Do people by default wish to have json/csv or HTML/PDF (consumable report) is preferable?

Reply:
I think that we are so used to reading json files that it may suffice